ArmTemplates/ServiceBus/servicebus-topic.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2018-05-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceBusNamespaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of an existing Service Bus namespace to add the topic to"
+            }
+        },
+        "serviceBusTopicName": {
+            "type": "string",
+            "metadata": {
+                "description": "Topic name to add to Service Bus"
+            }
+        },
+        "defaultTTL": {
+            "type": "string",
+            "defaultValue": "P90D",
+            "metadata": {
+                "description": "Default time to live (defaults to 90 days)"
+            }
+        },
+        "maxSize": {

```

Comment:
What is this the max size for?   
The size of the topic? 

Also, the description is incorrect.

Reply:
Yes, it was the size of the topic. I suggest renaming the parameter to better indicate what it is - maybe "TopicMaxSizeInMb" ?

Reply:
Updated to messageMaxSizeMb